x-pack/plugins/security_solution_serverless/server/task_manager/usage_reporting_task.ts

```diff
@@ -109,29 +104,35 @@ export class SecurityUsageReportingTask {
   ) => {
     // if task was not `.start()`'d yet, then exit
     if (!this.wasStarted) {
-      this.logger.debug('[runTask()] Aborted. Task not started yet');
       return;
     }
-
     // Check that this task is current
-    if (taskInstance.id !== this.getTaskId(taskInstance.taskType)) {
+    if (taskInstance.id !== this.taskId) {
       // old task, die
       throwUnrecoverableError(new Error('Outdated task version'));
     }
 
     const [{ elasticsearch }] = await core.getStartServices();
-
     const esClient = elasticsearch.client.asInternalUser;
+
     const lastSuccessfulReport = taskInstance.state.lastSuccessfulReport;
 
-    const usageRecords = meteringCallback({ esClient, lastSuccessfulReport });
+    const usageRecords = await meteringCallback({
+      esClient,
+      cloudSetup: this.cloudSetup,
+      logger: this.logger,
+      taskId: this.taskId,
+      lastSuccessfulReport,
+    });
+    this.logger.info(`received usage records: ${JSON.stringify(usageRecords)}`);
```

Comment:
I worry about the extra resources/cost we would incur here, especially if it's a lot of records. maybe `logger.debug` instead?